zmon_agent/discovery/kubernetes/cluster.py

```diff
@@ -16,6 +17,7 @@
 REPLICASET_TYPE = 'kube_replicaset'
 STATEFULSET_TYPE = 'kube_statefulset'
 DAEMONSET_TYPE = 'kube_daemonset'
+POSTGRESDB_TYPE = 'postgresql_database'
```

Comment:
It's supposed to be a generic one.  We set the `kube_cluster` property here, but the rest seems to be pretty generic and could be produced from AWS agent as well.  For that reason no `kube_` prefix in the type name.